tests/unit/project_loader/extensions/test_utils.py

```diff
@@ -675,6 +728,29 @@ def __init__(self, extension_name, yaml_data):
     return fixture_setup.FakeExtension("environment", ExtensionImpl)
 
 
+def _build_environment_extension_fixture():
+    class ExtensionImpl(Extension):
+        @staticmethod
+        def get_supported_bases() -> Tuple[str, ...]:
+            return ("core18",)
+
+        @staticmethod
+        def get_supported_confinement() -> Tuple[str, ...]:
+            return ("strict",)
+
+        def __init__(self, extension_name, yaml_data):
+            super().__init__(extension_name=extension_name, yaml_data=yaml_data)
+            self.root_snippet = {}
+            self.app_snippet = {}
+            self.part_snippet = {
+                "after": ["extension-part"],
+                "build-environment": [{"PATH": "$PATH:/bar"}, {"EXTKEY": "EXTVAL"}],
```

Comment:
assign $PATH/extensions-bin to make it easier to identify in the test.

Reply:
good idea, will do